modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting RolloutManager custom resource
+
+Uninstallation of {gitops-title} Operator does not remove any resources it created. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the RolloutManager custom resource (CR) and then uninstall the {gitops-title} Operator.
```

Comment:
```suggestion
To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the RolloutManager custom resource (CR).
```
The title suggests uninstallation of CR. Is it required to mention the uninstallation of the operator?

Reply:
Yes, because sometimes users might think that removing the Operator will remove all CR. Hence it is explicitly mentioned by @iam-veeramalla in his initial content.